style.css

```diff
@@ -20,32 +20,459 @@ ul {
   list-style: none;
 }
 
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px 155px;
+  position: fixed;
+  width: 100%;
+  height: 73px;
+  left: 0;
+  top: 0;
+  right: 0;
+  background: #fff;
+}
+
+.brand {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+#hamburger {
+  display: none;
+}
+
+#menu {
+  display: flex;
+  align-items: center;
+  gap: 32px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
+}
+
+#headline {
+  background-image: url("/images/background.svg");
```

Comment:
[Required]
- You're background images are currently not showing😭. I believe the problem may be the path you specified to your image file. Please update the path to use a relative path by adding a `.` before the file path, e.g.`"./images/background.svg"`🙏. This ensures that your image files are found regardless of the platform.

- Please also update the path for the background image of the contact form👍

- After adding the background image, please resize it as much as you can so the shapes areof similar sizes and in similar positions to the figma template 🖌 

**Note: The image may show on your computer, but it will not show anywhere else.**

Reply:
Fixed.